man/systemd.network.xml

```diff
@@ -220,8 +220,11 @@
           <command>systemd-networkd-wait-online</command>). When <literal>no</literal>, the network is ignored
           when determining the online state. When a minimum operational state and an optional maximum operational
           state are set, <literal>yes</literal> is implied, and this controls the minimum and maximum
-          operational state required for the network interface to be considered online.
-          Defaults to <literal>yes</literal>.</para>
+          operational state required for the network interface to be considered online.</para>
+
+          <para>Defaults to <literal>yes</literal>, except when <varname>ActivationPolicy=</varname>
+          is set to a value other than <literal>up</literal> or <literal>always-up</literal>,
+          this defaults to <literal>no</literal>.</para>
```

Comment:
"Defaults to yes, except when bar is set, this default to no." — this sentence is missing something. It'd be better to say "Default to foo, except when bar is set, when it defaults to no."

Reply:
i rewrote this a bit with the change in behavior re: `always-down` forcing `required_for_online = false`